3.evaluate_model/scripts/nbconverted/F1_scores.py

```diff
@@ -51,20 +49,28 @@
 # sorted so final models are shown before shuffled_baseline
 for model_path in sorted(models_dir.iterdir()):
     model = load(model_path)
-    # determine model/feature type/balance type from model file name
+    # determine model/feature type/balance/dataset type from model file name
     model_components = model_path.name.split("__")
-    # Older models only have 2 components, skip these
-    if len(model_components) == 2:
-        continue
     model_type = model_components[0]
     feature_type = model_components[1]
-    balance_type = model_components[2].replace(".joblib", "")
+    balance_type = model_components[2]
+    dataset_type = model_components[3].replace(".joblib", "")
```

Comment:
can you add comments here describing what "dataset_type" means

Reply:
Done!